tap_helpshift/streams.py

```diff
@@ -131,29 +138,18 @@ def sync(self, state):
 class IssueAnalytics(Stream):
     name = 'issue_analytics'
     url = 'analytics/issue'
-    key_properites = ['row_id']
+    key_properties = ['row_id']
     replication_method = 'INCREMENTAL'
     replication_key = 'updated_at'
 
-    def sync(self, state):
```

Comment:
Since this stream can't incrementally update anyway, I got rid of all the state/bookmark stuff here. Now this stream is only called from the Issues stream.